manifests/database/default_read_grant.pp

```diff
@@ -19,7 +19,7 @@
                   acl.defaclacl
                 FROM pg_default_acl acl
                 JOIN pg_namespace ns ON acl.defaclnamespace=ns.oid
-                WHERE acl.defaclacl::text ~ '.*\\\\\"${database_read_only_username}\\\\\"=r/${database_username}\\\".*'
+                WHERE array_to_string(acl.defaclacl, ' + ') ~ '(\"?)${database_read_only_username}\\1=r/(\"?)${database_username}\\2'
```

Comment:
```suggestion
                WHERE '@' || array_to_string(acl.defaclacl, '@') || '@' ~ '@(\"?)${database_read_only_username}\\1=r/(\"?)${database_username}\\2@'
```

Reply:
Hey!  I see you replaced the other occurrences of `<SPACE>+<SPACE>` with `@` but not this one.  Was it on purpose of just missing?  I think it's better to use the same construct each time for consistency.